manifests/07-operator.yaml

```diff
@@ -35,6 +35,8 @@ spec:
             requests:
               cpu: 10m
           env:
+            - name: RELEASE_VERSION
+              value: "0.0.1-snapshot"
```

Comment:
Given a payload substitution from the CVO, is there any way for @bparees (@adambkaplan now?) to go from this version to a level of code?  Navigate via the payload metadata?

Reply:
@deads2k Or reference a sha in the version number (4.0.z-sha)?

Reply:
The payload already identifies the git commit.  So `oc adm release extract --git PAYLOAD` will check out the right code, and `oc adm release info` will show the right. commit

Reply:
So all operators will have the same version, rather than unique versioning per operator?

Reply:
Correct.

Reply:
@smarterclayton What about operand version numbers? cluster-openshift-controller-manager-operator comes to mind - my understanding is the CVO sets the tag for the referenced images, ex origin-hypershift, origin-docker-builder, and origin-deployer